File: shanny-blog-java/blog-pojo/src/main/java/com/shanny/vo/AboutVO.java
package com.shanny.vo;

import com.shanny.enums.AboutEnum;
import io.swagger.v3.oas.annotations.media.Schema;
import jakarta.persistence.*;
import jakarta.validation.constraints.NotNull;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.format.annotation.DateTimeFormat;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Schema(description = "作者信息")
public class AboutVO {
    private Long id;
    private String name;
    private String introduce;
    private String avatar;
    private String tag;
    private String github;
    private String steam;
    private String web;
    private String biliBili;
    private Boolean isActive;
    private String other;
    private LocalDateTime createTime;
    private LocalDateTime updateTime;
}

File: shanny-blog-net/blog-pojo/Vos/AboutVO.cs
namespace blog_pojo.Vos
{
    public class AboutVO
    {
        public long? Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Introduce { get; set; } = string.Empty;
        public string Avatar { get; set; } = string.Empty;
        public string Tag { get; set; } = string.Empty;
        public string Github { get; set; } = string.Empty;
        public string Steam { get; set; } = string.Empty;
        public string Web { get; set; } = string.Empty;
        public string BiliBili { get; set; } = string.Empty;
        public bool? IsActive { get; set; }
        public string Other { get; set; } = string.Empty;
        public DateTime? CreateTime { get; set; }
        public DateTime? UpdateTime { get; set; }
    }
}
